add a glossary (#383)
ietf-rats-wg/draft-ietf-rats-corim: draft-ietf-rats-corim.md

in particular, CBOR diagnostic notation is defined in {{Section 8 of -cbor}}
and {{Section G of -cddl}}. Terms and concepts are always referenced as proper nouns, i.e., with Capital Letters.

This document uses the following terms:

{: vspace="0"}
Endorsed values:
: A set of characteristics of an Attester that do not appear in Evidence.
For example, Endorsed Values may include testing or certification data related to a hardware or firmware module.

> Endorsed Values are said to be "conditional" when they apply if other actual state matches Reference Values.

Reviewer: I confirmed that this is indeed the definition of Conditional Endorsed Values. However, this is super confusing. I think what we really mean is something on the lines below:

` Endorsed values are said to be "conditional", when they apply if the claim from the Attester Target Environment (ie. its actual state) matches Reference Values`.

Reviewer 2: Reference Values are corroborated and become accepted claims. Saying Reference Values gives the impression the condition applies to a Reference Value input.

> Endorsed Values are said to be "conditional" when they apply if other actual state matches Verifiers accepted Claims.

Author: ```suggestion
Endorsed Values are said to be "conditional" when they apply if Attester's actual state matches Verifier's accepted Claims.
```

See also {{Section 3 of -rats-endorsements}}.

Environment:
: A logical partition within an Attester.
The term "Target Environment" refers to the group of system security metrics that are reported through Evidence.
The term "Attesting Environment" refers to the entity that collects and cryptographically signs such security metrics.
See also {{Section 3.1 of -rats-arch}}.

Measurement:
: A value associated with specific security characteristics of an Attester that influences the trustworthiness of that Attester.
The object of a Measurement could be the invariant part of a firmware component loaded into memory during startup, a run-time integrity check (RTIC), a file system object, or a CPU register.
A measured object is part of the Attester's Target Environment.
Expected, or "golden," Measurements are compiled as Reference Values, which are used by the Verifier to assess the trust state of the Attester.
See also {{TNC.Arch}}, and Section 9.5.5 of {{TPM2.Part1}}.

Class ID:

> : An identifier for an Environment that is shared among similar Environments, such as those with the same hardware assembly.

Reviewer: ```suggestion
: An identifier for an Environment that is unique to a class or family, such as those with the same hardware assembly. It is shared among all the Environment instance.
```

> : An identifier for an Environment that is shared among similar Environments, such as those with the same hardware assembly.

Author: ```suggestion
: An identifier for an Environment that is shared among similar Environment instances, such as those with the same hardware assembly.
```

See also {{Section 4.2.4 of -eat}}.

Instance ID:
: An identifier of an Environment that is unique to that Environment instance, such as the serial number of a hardware module.
See also {{Section 4.2.1 of -eat}}.

Reference Values:
: A set of values that represent the desired or undesired state of an Attester.
Reference Values are compared against Evidence to determine the trustworthiness of the Attester.
Reference Values with matching Evidence produce "acceptable Claims."
See also {{Section 4.2 of -rats-arch}}, {{Section 8.3 of -rats-arch}}, and {{Section 2 of -rats-endorsements}}.

> Triple:

Reviewer 3: Can you make this section more compact?

> : A term derived from Resource Description Framework (RDF) to mean a statement expressing a relationship between a subject and an object resource.

Author: ```suggestion
: A term derived from the Resource Description Framework (RDF) to mean a statement expressing a relationship between a subject and an object resource.
```

The nature of the relationship between subject and object is expressed via a predicate.

> Unlike RDF, in CoRIM, the predicate of the triple is implicit and is encoded in the triple's name/codepoint.

Author: ```suggestion
In CoRIM, unlike RDF, the predicate of the triple is implicit and is encoded in the triple's name/codepoint.
```

> In CoRIM, triples typically represent assertions made by the CoRIM author regarding Attesting or Target Environments and their security features, such as Measurements and cryptographic key material.

Author: ```suggestion
CoRIM triples typically represent assertions made by the CoRIM author regarding Attesting or Target Environments and their security features, such as Measurements and cryptographic key material.
```
